Bluepill-cli/Bluepill-cli/Simulator/SimulatorRunner.m

```diff
@@ -230,6 +301,20 @@ - (BOOL)installApplicationAndReturnError:(NSError *__autoreleasing *)error {
     return YES;
 }
 
+- (BOOL)uninstallApplicationAndReturnError:(NSError *__autoreleasing *)error {
```

Comment:
I'm not sure what's going on, but it seems we're reviewing something that doesn't have my latests changes.